src/MSBuild/LiveLogger/TerminalBuffer.cs

```diff
@@ -57,10 +59,13 @@ public TerminalBufferLine(string text, bool shouldWrapLines)
     internal class TerminalBuffer
     {
         private static List<TerminalBufferLine> Lines = new();
+        public static string FooterText = string.Empty;
         public static int TopLineIndex = 0;
         public static string Footer = string.Empty;
         internal static bool IsTerminated = false;
         internal static bool ShouldRerender = true;
+        internal static int FinishedProjects = 0;
```

Comment:
I know this is only using the same practise already used in this component - but it would be nice to reserve fields fro strictly private usage, other access should be via properties or methods.

But it is beyond the scope of this PR. I'm thinking about creating an item for improving styling and coding standards in LiveLogger and doing those changes in one shot in separate PR - thoughts?